source/_integrations/buienradar.markdown

```diff
@@ -6,125 +6,108 @@ ha_category:
+- `irradiance`: "Sun intensity in Watt per square meter ([W/m2](https://en.wikipedia.org/wiki/W/m2))."
+- `rainlast24hour`: The rail over the last 24 hours (in mm).
+- `rainlasthour`: The rail over the last hour (in mm). 
+- `temperature_1d` `temperature_2d` `temperature_3d` `temperature_4d` `temperature_5d`: "The forecasted temperature (in [C](https://en.wikipedia.org/wiki/Celsius))."
+- `mintemp_1d` `mintemp_2d` `mintemp_3d` `mintemp_4d` `mintemp_5d`: "The forecasted minimum temperature (in [C](https://en.wikipedia.org/wiki/Celsius))."
+- `rainchance_1d` `rainchance_2d` `rainchance_3d` `rainchance_4d` `rainchance_5d`: The forecasted chance for rain (%).
+- `sunchance_1d` `sunchance_2d` `sunchance_3d` `sunchance_4d` `sunchance_5d`: The forecasted chance for sun (%).
+- `rain_1d` `rain_2d` `rain_3d` `rain_4d` `rain_5d`: "The forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimetre); the average of minrain_1d and maxrain_1d."
+- `minrain_1d` `minrain_2d` `minrain_3d` `minrain_4d` `minrain_5d`: "The minimum forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimetre)."
+- `maxrain_1d` `maxrain_2d` `maxrain_3d` `maxrain_4d` `maxrain_5d`: "The maximum forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimetre)."
+- `windazimuth_1d` `windazimuth_2d` `windazimuth_3d` `windazimuth_4d` `windazimuth_5d`: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise. (derived from winddirection_1d)
+- `winddirection_1d` `winddirection_2d` `winddirection_3d` `winddirection_4d` `winddirection_5d`: "Where the wind will be coming from: N (North), Z (south), NO (North-East), etc."
+- `windforce_1d` `windforce_2d` `windforce_3d` `windforce_4d` `windforce_5d`: "The expected windforce in [Bft](https://en.wikipedia.org/wiki/Beaufort_scale)."
+- `windspeed_1d` `windspeed_2d` `windspeed_3d` `windspeed_4d` `windspeed_5d`: "The expected windspeed in [m/s](https://en.wikipedia.org/wiki/M/s) (derived from windforce_1d)."
+- `conditioncode_1d` `conditioncode_2d` `conditioncode_3d` `conditioncode_4d` `conditioncode_5d`: Symbol and condition code of the expected condition.
+- `condition_1d` `condition_2d` `condition_3d` `condition_4d` `condition_5d`: Symbol and expected condition.
+- `conditiondetailed_1d` `conditiondetailed_2d` `conditiondetailed_3d` `conditiondetailed_4d` `conditiondetailed_5d`: Symbol and detailed expected condition.
+- `conditionexact_1d` `conditionexact_2d` `conditionexact_3d` `conditionexact_4d` `conditionexact_5d`: Symbol and full expected condition (in English).
+- `symbol_1d` `symbol_2d` `symbol_3d` `symbol_4d` `symbol_5d`: Symbol and full expected condition (in Dutch).
+
+## Setting options
+
+To configure options for `buienradar` integration go to **Configuration** >> **Integrations** and press **Options** on the Buienradar card.
 
 {% configuration %}
```

Comment:
Use `configuration_basic` instead, that was we can remove all `required`, `default` and `type` parameters (as they are handled and shown in the UI).